### DISCUSSION TITLE
using bun for the rendering engine

### DISCUSSION THREAD

**Discussion by Fanna1119** [silviogutierrez/reactivated#396] [Ideas]
thoughts on using bun for the rendering engine?
alongside https://react.dev/reference/react-dom/server/renderToReadableStream 
https://bun.sh/guides/ecosystem/ssr-react

**Comment by silviogutierrez** [comment]
I'd love to see a POC of this. Apparently bun is faster at rendering.

If you look at `render.mts` it should be fairly simple to try.

**Reply by Fanna1119** [reply]
I'll see if I can put something together.
